From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dyscorv <174674743+dyscorv@users.noreply.github.com>
Date: Fri, 23 Aug 2024 00:00:00 +0000
Subject: [PATCH] subst-executables


diff --git a/animatediff/nodes_deprecated.py b/animatediff/nodes_deprecated.py
index 50fe5c9..f6330e9 100644
--- a/animatediff/nodes_deprecated.py
+++ b/animatediff/nodes_deprecated.py
@@ -154,7 +154,7 @@ class AnimateDiffCombineDEPR:
     ffmpeg_warning_already_shown = False
     @classmethod
     def INPUT_TYPES(s):
-        ffmpeg_path = shutil.which("ffmpeg")
+        ffmpeg_path = "@ffmpeg@"
         #Hide ffmpeg formats if ffmpeg isn't available
         if ffmpeg_path is not None:
             ffmpeg_formats = ["video/"+x[:-5] for x in folder_paths.get_filename_list(Folders.VIDEO_FORMATS)]
@@ -258,7 +258,7 @@ class AnimateDiffCombineDEPR:
             )
         else:
             # Use ffmpeg to save a video
-            ffmpeg_path = shutil.which("ffmpeg")
+            ffmpeg_path = "@ffmpeg@"
             if ffmpeg_path is None:
                 #Should never be reachable
                 raise ProcessLookupError("Could not find ffmpeg")
