From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: dyscorv <174674743+dyscorv@users.noreply.github.com>
Date: Sat, 7 Dec 2024 00:00:00 +0000
Subject: [PATCH] fix-version


diff --git a/server.py b/server.py
index 3611880..466cf58 100644
--- a/server.py
+++ b/server.py
@@ -42,12 +42,11 @@ async def send_socket_catch_exception(function, message):
         logging.warning("send error: {}".format(err))
 
 def get_comfyui_version():
+    return "@version@"
     comfyui_version = "unknown"
     repo_path = os.path.dirname(os.path.realpath(__file__))
     try:
-        import pygit2
-        repo = pygit2.Repository(repo_path)
-        comfyui_version = repo.describe(describe_strategy=pygit2.GIT_DESCRIBE_TAGS)
+        pass
     except Exception:
         try:
             import subprocess
